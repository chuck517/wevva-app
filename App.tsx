import React from 'react';
import AppProvider from './AppContext';
import Main from './components/Main';

export default function App() {
  return (
    <AppProvider>
      <Main />
    </AppProvider>
  );
}
