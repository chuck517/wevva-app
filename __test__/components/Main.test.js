import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../components/Main';


it('renders without crashing', () => {
  const rendered = renderer.create(<App />).toJSON();
  expect(rendered).toBeTruthy();
});

it('test against snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});