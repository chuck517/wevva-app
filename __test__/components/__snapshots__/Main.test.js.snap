// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly across screens 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "paddingTop": 80,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": "100%",
          "justifyContent": "center",
          "width": "100%",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#fff"
        hidesWhenStopped={true}
        size="large"
      />
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 17,
          }
        }
      >
        Finding your location...
      </Text>
    </View>
  </View>
</View>
`;

exports[`test against snapshot 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "paddingTop": 80,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": "100%",
          "justifyContent": "center",
          "width": "100%",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#fff"
        hidesWhenStopped={true}
        size="large"
      />
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 17,
          }
        }
      >
        Finding your location...
      </Text>
    </View>
  </View>
</View>
`;
